module2/homework/addEven/addEven.hpp

```diff
@@ -2,7 +2,9 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int tmp;
+    for(auto number : numbers){
```

Comment:
```suggestion
    for(const auto &number : numbers){
```
as you don't modify the `number`